docs/Child-friendly-EmulationStation.md

```diff
@@ -13,23 +13,23 @@ These modes allow administrators to lock down (part of) the system, so that naiv
 ***
 
 # UI modes
-To allow different levels of system protection, user modes were introduced with different sets of functionalities. These modes are Full, Kiosk and Kid. Generally speaking, kids do not need to edit system settings, nor will visiting friends. The full set of options also complicates the UI, and the kid and kiosk modes hide these when they are not needed.
+To allow different levels of system protection, user modes have been introduced to allow the user to restrict certain settings and files from use. The three new modes are Full, Kiosk and Kid. In general, visiting friends and children do not need to modify system settings. The full set of options also complicates the UI, and the kid and kiosk modes hide these for a simpler experience.
```

Comment:
There are only 2 'new' modes, the _Full_ mode is the default/built-in UI.

Reply:
Good catch, fixed with 647c65e998bdc6c4ab84289e755dc773d8313aa7